src/main/resources/org/broadinstitute/dsde/sam/liquibase/changesets/20230203_last_quota_error.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="dummy"
+                   xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+
+    <changeSet logicalFilePath="dummy" author="dvoet" id="last_quota_error">
+        <createTable tableName="SAM_LAST_QUOTA_ERROR">
```

Comment:
nit - "last quota error" is a little generic, there are lots of quotas.  Ideally this table would be named more precisely or better yet, maybe there's an additional column for `QUOTA_TYPE` or `QUOTA_NAME`.  We can slay that dragon when it comes for our treasure though.